src/Symfony/Component/Messenger/Handler/HandlerInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Handler;
+
+/**
+ * Handlers can implements this interface. This allows them to be auto-configured and to
+ * handle multiple requests.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+interface HandlerInterface
+{
+    /**
+     * Return a list of messages to be handled.
+     *
+     * It returns a list of messages like in the following example:
+     *
+     *     return [
+     *         FirstMessage::class,
+     *         SecondMessage:class
+     *     ];
```

Comment:
Probably the first example should just show the simplest & most common use-case (1 message):

```php
return [SomeMessage::class];
```